select_assignment2_flicks/ViewController.swift

```diff
@@ -188,6 +190,61 @@ extension ViewController: UITableViewDelegate, UITableViewDataSource {
         }
         let movie = movies[indexPath.row]
         cell.setData(movie)
+        cell.leftButtons = [MGSwipeButton(title: "Favorite", backgroundColor: UIColor.blackColor()) { (cell) -> Bool in
+            self.movies[indexPath.row].isFavorite = true
+            tableView.reloadData()
+            return true
+            }]
+        cell.leftExpansion.buttonIndex = 0
+        
+        cell.rightButtons = [ MGSwipeButton(title: "Share", backgroundColor: UIColor.blackColor()) { (cell) -> Bool in
```

Comment:
setting up these buttons might be nicely refactored into a method. it feels a bit long